### PR TITLE
Feat: skills

### LINKED ISSUE #14: Implement API route to return a specific Skill
On the `/resume/skill` route, you should

 

- [x] Return a specific Skill when a GET request is made with an index as an input. The Skill should be in a JSON format.

### LINKED ISSUE #15: Implement API route to return all "Skills"
On the `/resume/skill route`, you should:

- [x]  Return the Skill list when a GET request is made. The return should be in a list format.

### LINKED ISSUE #16: Implement API route to add new "Skills"
On the` /resume/skill `route, you should:

- [x]  Return the index position of the new Skill in the Skill list when a POST request is made

### PR DESCRIPTION
**Pull request by DeleMike**
- Implemented API route to get a skill based on index value
- Implement API route to get all skills
- Implement API route to add new skill

Closes #14 , closes #15, closes #16

### CODE REVIEW

**Comment by DeleMike** [comment]
Hi @dtemir, please review.

File: app.py [hunk 1]
```diff
@@ -119,15 +119,40 @@ def skill():
     Handles Skill requests
     '''
     if request.method == 'GET':
-        return jsonify({})
+        return data['skill'], 200
```
Comment (dtemir) [inline app.py:122]: Please review issue #12 again. You must use return a skill given an index

Comment (DeleMike) [reply app.py:122]: I actually return a skill given an index in the `a_skill` function

Comment (dtemir) [reply app.py:122]: My bad! Working from my phone so I overlooked that. Is there a way to combine the two into one function? Do you have any plans for the initial function?

Comment (DeleMike) [reply app.py:122]: Okay, I will look into this ASAP

**Comment by DeleMike** [comment]
@dtemir, could you review this again? 
`test_add_skill` in `test_pytest.py` is passing too.

**Comment by dtemir** [comment]
Great. Let's this merged

**Comment by DeleMike** [comment]
Let's go!

**Pull request merged**